runtime/v2/model/src/main/java/com/walmartlabs/concord/runtime/v2/ConcordJsonSchemaGenerator.java

```diff
@@ -68,7 +68,7 @@ public static JsonNode generate() {
         switchStep.set("additionalProperties", switchDefault);
 
         // remove invalid required primitive attributes
-        removeRequired(path(jsonSchema, "definitions/ProcessDefinitionConfiguration"), "debug");
+        removeRequired(path(jsonSchema, "definitions/ProcessDefinitionConfiguration"), "debug", "parallelLoopParallelism");
```

Comment:
this is just some extra cleanup right?

Reply:
yep. current version of JsonSchemaGenerator lib marks booleans as required fields.